### PR TITLE
data: Hasnaat-Iftikhar.json

### PR DESCRIPTION
**Pull request by Hasnaat-Iftikhar**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2884"></a>

### CODE REVIEW

File: hasnaat-iftikhar.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
+  "bio": "FullStack developer and Elevating my skills in Cloud!",
+  "avatar": "https://lh3.googleusercontent.com/oL7jB41g1IoOM1VCwAckG6TVF-cPZkzwk0kRK7aJ47IrEvXj1elbek964WTMhICi-mpRHBAWw34o4F62VzeozfBd6zKOVMHrtAD-YMVwPrSZuv17uQKZ28pYOMNMIx1JWuB3Gn9PMcoD__Bu9uA6ov39JmBKKwc2vtQ7NGGh4goGtPpK-ga0aljTWfWBthR73xbt_nGf0l643csyFY0LXStEoFmZBrFOI8Pa0A5NjAFeFpsB_PRuToFAqmZAgwaKJNOvzWtPFg6eTL120VYORN0vW15Ezk9KgqbB0pFunX1ibSR--jAYFPeTzNTavT8Dnmk3pYaUjmgumLxvEKR6MrazPx_GK2R76PzByeb7PvAGGoEAb_CnCKsubbCS7zbKIPUhTumlVriMysay1psEhYAtKz3e9J_GQQOuwBBbtCoF5oykbwQeSlXwxCLib8PfIPLqSpQFbnmpChEHUlHRsVKpffWWcESNlPdJ5_3szaL5OmNPakRkpxsoAgTk7IeIRMdMMAzXLTwALLb-g23G3Rrld3RDLsRI_5xlOvAiY11WyU8vcoUx4UFEiJ2ddmEcmmhZwBliVA7HflNr9JWE-gRzZK1NglIOZm4QtXSeVDRyMGBxJPtb5w-hou9j6fFDVdmx90t2ww3aK7Vocy2OiWne97_aQdeELMzMvxkIeZZw_53181v4lYtyTSVUYiFc0EGhkJc1W1jPwJeb9glqV2-t8vO7uQDwFIocWGGP2tBvJucu8-0QpvGLYi_UhIpkREV-uAvXa1_plgDhSVT-cZxqy_REJB1KYjOzd1dNT6Z0ZpI6VgtZ9TbSVtx_zd7SBdyTblsEDmi1ZieanXgUPIc8PbcyqyoCMYr_U8I5_D8726j_zJd0anfxqxto69uN-5AXX46bONY1DQ1w6uoNcHtxAQDyyoCw4uKtULAEznz-H40gcBvlda800dBOdhbrK80Kcg35Y3se9_mT0w4D7THUxUtThEe5oYNN0n1qR8zrUVi9qL32lw=s625-no?authuser=0",
```
Comment (Aadarsh805) [inline hasnaat-iftikhar.json:6]: use this for the avatar link: `https://github.com/Hasnaat-Iftikhar.png`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by dan-mba — CHANGES_REQUESTED** [review]
Please move your profile to the data directory

**Comment by eddiejaoude** [comment]
If you have any questions about the comments feel free to ask here and someone will help. If you want to have a realtime chat we also have an EddieHub Discord you can join http://discord.eddiehub.org

**Review by EmmaDawsonDev — DISMISSED** [review]
Please update the name of your file to match your github username exactly (case-sensitive). Let us know if you need any help.

**Review by Hasnaat-Iftikhar — COMMENTED** [review]
Avatar image link changed successfully

File: hasnaat-iftikhar.json (hunk 1 above)
Comment (Hasnaat-Iftikhar) [reply hasnaat-iftikhar.json]: The replaced the avatar link successfully

**Comment by EmmaDawsonDev** [comment]
> Avatar image link changed successfully

Great! Now all that is left is to move your file into the data folder and then rename it to match your github username.

**Review by SaraJaoude — APPROVED** [review]
I will merge and fix the file path and GitHub username case mismatch in a temporary branch

**Pull request merged**